client/login/magic-login/request-login-email-form.jsx

```diff
@@ -165,7 +165,7 @@ class RequestLoginEmailForm extends Component {
 					<p className="magic-login__form-sub-header">
 						{ ! hideSubHeaderText &&
 							translate(
-								'Get a link sent to the email address associated with your account to log in instantly without your password.'
+								'We’ll send you an email with a login link that will log you in right away.'
```

Comment:
We will need an `isJetpackLogin` flag (or a similar identifier) to differentiate wordings from WP.com.

Reply:
Yup, great catch. I'll update this and review on the CSS comment as well. Thanks for reviewing @wellyshen!

Reply:
@wellyshen  - After discussing with @simison, we decided to just leave this new text in place for both places.